frontend/package.json

```diff
@@ -20,20 +21,16 @@
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
     "@heroicons/react": "^2.1.1",
-    "@langchain/community": "^0.0.27",
-    "@langchain/core": "^0.1.26",
-    "@langchain/openai": "^0.0.14",
-    "@langchain/weaviate": "^0.0.1",
     "@rjsf/chakra-ui": "^5.17.1",
     "@rjsf/core": "^5.17.1",
     "@rjsf/utils": "^5.17.1",
     "@rjsf/validator-ajv8": "^5.17.1",
     "@tanstack/react-query": "^5.28.4",
-    "@types/dompurify": "^3.0.5",
     "@types/marked": "^5.0.1",
     "@types/node": "20.4.9",
     "@types/react": "18.2.20",
     "@types/react-dom": "18.2.7",
+    "@types/uuid": "^9.0.8",
```

Comment:
While we're at it, we should move all types prefixed with `@types/` to `devDependencies`